adm_program/languages/de.xml

```diff
@@ -927,6 +927,10 @@
     <string name="PRO_WANT_DELETE_PHOTO">Soll das ausgewählte Foto wirklich gelöscht werden?</string>
     <string name="PRO_YEARS">Jahre</string>
     <!-- Phrases only in Relationssmodule -->
+    <string name="REL_EDIT_USER_BACKWARD">Benutzerprofil der rückwärtigen Beziehung bearbeiten</string>
+    <string name="REL_EDIT_USER_DESC">Ist diese Einstellung gesetzt, dann darf der Benutzer die Profile der anderen Benutzer bearbeiten zu denen er in Beziehung steht. Bei einer asymmetrischen Beziehung wird das Recht separat je Beziehungsrichtung vergeben, bei einer symmetrischen Beziehung dürfen beide Benutzer das jeweils andere Profil bearbeiten und bei einer unidirektionalen Beziehung darf nur der Benutzer mit der hinterlegten Beziehung das andere Profil bearbeiten.</string>
+    <string name="REL_EDIT_USER_FORWARD">Benutzerprofil der vorwärtigen Beziehung bearbeiten</string>
```

Comment:
Sounds strange for me: Might be changed to e.g "Benutzerprofil der Folgebeziehung bearbeiten"